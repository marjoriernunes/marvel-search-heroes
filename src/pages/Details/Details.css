.header-details {
    display: flex;
    flex-direction: row;
}

.details-page {
    background-color: #f2f2f2;
}

.details-page section {
    text-align: center;
    margin: 0 2rem;
}

.footer-bar {
    margin-top: 6rem;
}