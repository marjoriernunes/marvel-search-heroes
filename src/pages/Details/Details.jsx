import './Details.css';
import Icon from '../../assets/Icons';
import Logo from '../../assets/logo_menor.svg';
import SearchBar from '../../components/searchbar/SearchBar';
import Footer from '../../components/footer/Footer';
// import { useParams } from 'react-router-dom';

const Details = () => {
    // const params = useParams();
    // console.log(params?.id);

    const teste = () => {
        console.log('foi');
    }
    return (
        <>
            <div className="details-page">
                <section id="header" className="header-details">
                    <img className="home-logo" src={Logo} alt="Marvel Search Heroes"/>
                    <SearchBar styleClass="details-search" sendData={(value) => {teste(value)}}/>
                </section>
                <section id="character-content" className="header-details">
                    <div>
                        <h2>NAME</h2>
                        <Icon name="favorito_01" width="20px" height="20px"/>
                    </div>
                    <div>
                        IMAGE
                    </div>
                </section>
                <section id="character-comics">
                    <div>COMIC BOOKS</div>
                </section>
                <Footer className="footer-bar"/>
            </div>
        </>
    );
}

export default Details;