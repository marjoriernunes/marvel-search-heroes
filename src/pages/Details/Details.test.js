import {render} from '@testing-library/react';
import Details from './Details';

test('renders correctly', () =>{
    const utils = render(<Details/>)
    expect(utils.container).toMatchSnapshot()
});