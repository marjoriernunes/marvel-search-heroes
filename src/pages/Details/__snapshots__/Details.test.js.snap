// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="details-page"
  >
    <section
      class="header-details"
      id="header"
    >
      <img
        alt="Marvel Search Heroes"
        class="home-logo"
        src="logo_menor.svg"
      />
      <div
        class="left-icon details-search"
      >
        <svg
          height="20px"
          width="20px"
        >
          <use
            xlink:href="icons.svg#icon-ic_busca"
          />
        </svg>
        <input
          placeholder="Procure por herÃ³is"
          type="text"
        />
      </div>
    </section>
    <section
      class="header-details"
      id="character-content"
    >
      <div>
        <h2>
          NAME
        </h2>
        <svg
          height="20px"
          width="20px"
        >
          <use
            xlink:href="icons.svg#icon-favorito_01"
          />
        </svg>
      </div>
      <div>
        IMAGE
      </div>
    </section>
    <section
      id="character-comics"
    >
      <div>
        COMIC BOOKS
      </div>
    </section>
    <div
      class="footer-bar"
    />
  </div>
</div>
`;
